logger/src/main/java/com/orhanobut/logger/MultiLogAdapter.java

```diff
@@ -0,0 +1,60 @@
+package com.orhanobut.logger;
+
+import java.util.List;
+
+/**
+ * Created by ronaldo pace on 1/5/17. https://github.com/budius
+ * Simple LogAdapter implementation that re-distributes the messages to other implementations.
+ */
+public class MultiLogAdapter implements LogAdapter {
```

Comment:
I like it. As I mentioned earlier, I tried to disrupt as minimum as possible the existing API. But for sure `add` is a more elegant solution.